cppcache/src/PdxRemoteWriter.hpp

```diff
@@ -22,11 +22,13 @@
 
 #include "PdxLocalWriter.hpp"
 
+#include <geode/internal/geode_globals.hpp>
+
 namespace apache {
 namespace geode {
 namespace client {
 
-class PdxRemoteWriter : public PdxLocalWriter {
+class APACHE_GEODE_EXPORT PdxRemoteWriter : public PdxLocalWriter {
```

Comment:
Please don't export this, we can't change the ABI and anyway we don't want any more publicly-accessible things than we already have(!).

Reply:
This class is exported to allow testing. Thing is I thought there are no issues from the ABI-side if you added new symbols, rather than whenever you changed symbols. Am I wrong?